glide-core/redis-rs/redis/src/cluster_async/mod.rs

```diff
@@ -363,6 +366,35 @@ where
                 Response::ClusterScanResult(_, _) => unreachable!(),
             })
     }
+    /// Replace credentials of inner connection
+    pub async fn reset_connection_password(&mut self, password: String) -> RedisResult<Value> {
+        self.route_operation_request(Operation::UpdateConnectionPassword { password })
+            .await
+    }
+
+    /// Routes an operation request to the appropriate handler.
+    async fn route_operation_request(
+        &mut self,
+        operation_request: Operation,
+    ) -> RedisResult<Value> {
+        let (sender, receiver) = oneshot::channel();
+        self.0
+            .send(Message {
+                cmd: CmdArg::OperationRequest { operation_request },
+                sender,
+            })
+            .await
+            .map_err(|_| RedisError::from(io::Error::from(io::ErrorKind::BrokenPipe)))?;
+
+        receiver
+            .await
+            .unwrap_or_else(|_| Err(RedisError::from(io::Error::from(io::ErrorKind::BrokenPipe))))
+            .map(|response| match response {
+                Response::Single(values) => values,
+                Response::Multiple(_) => unreachable!(),
+                Response::ClusterScanResult(_, _) => unreachable!(),
```

Comment:
Can you add a comment why unreachable?
```suggestion
                _ => unreachable!(),
```

Reply:
True